src/hooks/screens/useBookRegistration.ts

```diff
@@ -20,8 +21,9 @@ const SEARCH_DEBOUNCE_DELAY = 500;
  * 書籍登録画面で使用する検索・選択・登録機能を統合管理するフック
  */
 export const useBookRegistration = () => {
+  const { path: _, search } = useLocalSearchParams<{ path: string; search?: string }>();

```

Comment:
searchParamにsearchを受け取ってたら、それを検索の初期値にセットして一覧取得している
```suggestion
  const { path: _, search } = useLocalSearchParams<{ path: string; search?: string }>();

```